include/nfc/t4t/hl_procedure.rst

```diff
@@ -1,34 +1,33 @@
 .. _nfc_t4t_hl_procedure_readme:
 
-NFC T4T High Level Procedure
-############################
+High-level procedure
```

Comment:
Is this the correct name? "High-level" does not contain any information on what this library does.

Reply:
I changed the topic type to Type 4 Tag procedures now, which has a bit more content than high level procedure. :)
This should be followed up though in another PR to also rename the files, API functions, and update the API doc.